cli/src/input.rs

```diff
@@ -66,6 +86,40 @@ impl<C: clap::Args + Customize> Prepare for InputOptions<C> {
             program.add_import_paths(nickel_path.split(':'));
         }
 
+        #[cfg(feature = "package-experimental")]
+        {
+            let manifest_path = self.manifest_path.clone().or_else(|| {
+                // If the manifest path isn't given, where should we start looking
+                // for it? If there's only one file, we start with its parent directory.
+                // If there are no files, we take the current directory.
+                //
+                // If there are multiple files, finding a good heuristic
+                // is harder. For now, we take the parent directory if it's
+                // unique and otherwise we require an explicit manifest
+                // path.
```

Comment:
One possibility would be to look for the common ancestor. But I can see how it might be wrong in some cases (if the files are far apart), so playing safe by explicitly requiring a manifest path sounds good :+1: